poetry.lock

```diff
@@ -1,42 +1,41 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
```

Comment:
Could you update poetry?

Reply:
fixed on https://github.com/m3dev/gokart/pull/417/commits/ea7525b6d6a082a4e71e90e6f3a9bf03672c5538